.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
```

Comment:
Should we test only on 0.10? We do run 0.10 in production, but testing in 0.12 also is going to save us pain when our production servers update.